tools/kdc.json

```diff
@@ -6,30 +6,24 @@
   "container": {
     "type": "MESOS",
     "docker": {
-      "image": "mesosphere/kdc:latest",
+      "image": "nvaziri/kdc:dev",
       "forcePullImage": true
-    },
-    "portMappings": [
-      {
-        "containerPort": 88,
-        "protocol": "tcp",
-        "labels": {"VIP_0": "kdc:88"}
-      },
-      {
-        "containerPort": 88,
-        "protocol": "udp",
-        "labels": {"VIP_0": "kdc:88"}
-      },
-      {
-        "containerPort": 8000,
-        "protocol": "tcp",
-        "labels": {"VIP_0": "kdc:8000"}
-      }
-    ]
+    }
   },
   "networks": [
     {
-      "mode": "container/bridge"
+      "mode": "host"
+    }
+  ],
+  "portDefinitions": [
+    {
+      "port": 2500,
```

Comment:
Note that this isn't the actual port KDC listens on as the application port, 2500 just maps to a dynamically assigned port at runtime.